domain_director/domain.py

```diff
@@ -66,16 +65,17 @@ def decide_node_domain(node_id, polygons, lat=None, lon=None, accuracy=None, max
 
     return domain, criteria
 
-
-def get_node_domain(node_id, polygons, wifis=None, api_key="test", lat=None, lon=None, accuracy=None,
+def update_node_domain(node_id, polygons, wifis=None, api_key="test", lat=None, lon=None, accuracy=None,
                     default_domain=None,
-                    max_accuracy=250, treshold_distance=0, switch_time=-1, migrate_only_vpn=False):
+                    max_accuracy=250, treshold_distance=0, default_switch_time=-1, migrate_only_vpn=False):
     is_vpn_only = False
     mesh_id = Node.get_mesh_id(node_id)
     if mesh_id is None:
         domain = default_domain
+        out_switch_time = default_switch_time
```

Comment:
I dont' understand what "out" means in that case.

Reply:
`default_switch_time` could also stay as just `switch_time` param, that gets overwritten if certain required.

Reply:
I changed the name to switch_time, good idea.
regarding the parameter: I think it would be more clear to leave the parameter name default_switch_time. When calling the function with the parameter named default_* you would probably expect that the value could be changed within the function.